packages/nestjs-user/src/listeners/invitation-accepted-listener.ts

```diff
@@ -63,6 +63,8 @@ export class InvitationAcceptedListener
         throw new UserNotFoundException();
       }
 
+      // TODO: UPDATE PASSWORD how to get roles here?
+      // should return on user lookup?
```

Comment:
need to check if they exist on the object. invitation module should be forwarding the entire payload